packages/docs/package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "@defichain/metachain-docs",
+  "name": "@defimetachain/docs",
   "version": "0.0.0",
```

Comment:
Should this be higher than 0.0.0?
I am not really familiar with the software versioning used in cake. Do you have a link to the software versioning doc used in cake?

Reply:
We don't have this documented yet, I'm working on the release model for this repo soon.

https://github.com/JellyfishSDK/jellyfish/blob/main/CONTRIBUTING.md#publishing

The TLDR is it will follow very closely to what was done at Jellyfish, where the release and the version are determined by the release drafter and annotated on release since we're doing trunk-based development.

It's related to this PR https://github.com/DeFiCh/metachain/pull/47 where everything else is heavily automated, we just focus on pushing implementation and testing it.

Reply:
Thanks!